openassessment/templates/openassessmentblock/oa_rubric.html

```diff
@@ -7,29 +7,35 @@
+                <button class="ui-slidable" aria-expanded="true" id="oa_rubric__{{ rubric_type }}__{{ submission.uuid|default:'dummy_uuid' }}__{{ criterion.order_num }}" aria-controls="oa_rubric__{{ rubric_type }}__{{ submission.uuid|default:'dummy_uuid' }}__{{ criterion.order_num }}__content">
                     <span class="icon fa fa-caret-right" aria-hidden="true"></span>
                     <span id="{{ rubric_type }}__assessment__rubric__prompt--{{ criterion.order_num }}" class="ui-slidable__control__copy question__title__copy">{{ criterion.prompt }}</span>
                     <span class="label--required sr">({% trans "Required" %})</span>
                 </button>
             </div>
 
-            <div class="ui-slidable__content" aria-labelledby="oa_rubric__{{ rubric_type }}__{{ submission.uuid }}__{{ criterion.order_num }}" id="oa_rubric__{{ rubric_type }}__{{ submission.uuid }}__{{ criterion.order_num }}__content">
+            <div class="ui-slidable__content" aria-labelledby="oa_rubric__{{ rubric_type }}__{{ submission.uuid|default:'dummy_uuid' }}__{{ criterion.order_num }}" id="oa_rubric__{{ rubric_type }}__{{ submission.uuid|default:'dummy_uuid' }}__{{ criterion.order_num }}__content">
                 <div class="question__answers">
                     <div role="group" aria-labelledby="{{ rubric_type }}__assessment__rubric__prompt--{{ criterion.order_num }}">
                         {% for option in criterion.options %}
                         <div class="answer">
                             <div class="wrapper--input">
-                                <input type="radio"
-                                       data-criterion-name="{{ criterion.name }}"
-                                       class="answer__value rubric_{{ criterion.order_num }}_{{ option.order_num }}"
-                                       id="{{ rubric_type }}__assessment__rubric__question--{{ criterion.order_num }}__{{ option.order_num }}__{{ xblock_id }}"
-                                       value="{{ option.name }}"
-                                       name="{{ rubric_type }}__assessment__rubric__question--{{ criterion.order_num }}"
-                                       aria-describedby="{{ rubric_type }}__assessment__rubric__meta--{{ criterion.order_num }}__{{ option.order_num }}__{{ xblock_id }}"/>
-                                <label for="{{ rubric_type }}__assessment__rubric__question--{{ criterion.order_num }}__{{ option.order_num }}__{{ xblock_id }}"
-                                       class="answer__label"
-                                       >{{ option.label }}</label>
+                                {% if rubric_type == 'read_only' %}
+                                    <label for="{{ rubric_type }}__assessment__rubric__meta--{{ criterion.order_num }}__{{ option.order_num }}__{{ xblock_id }}"
```

Comment:
easy to miss, but the reason I don't just gate off the `<input>` and leave the `<label>` is because here I also point this label's `for` at the proceeding `<div>` rather than the `<input>`

(i say easy to miss because when I came back to this i was totally confused why i did this)